package.json

```diff
@@ -54,7 +54,7 @@
     "semantic-release": "^17.1.2"
   },
   "engines": {
-    "node": ">= 6"
+    "node": ">= 10.14.2"
```

Comment:
If you are bumping the min node version to 10.14.2, this would technically be a breaking change, especially since you are dropping support of Node 8.

Reply:
Yes this would require a major release. Thus my question around bumping to requiring Node.js 12.13 immediately to (hopefully) be able to upgrade to jest-mtcher-utils 27 without another breaking change.